tests/components/charts.test.ts

```diff
@@ -1307,6 +1310,42 @@ describe("figures", () => {
       ]);
     });
   });
+
+  test("Can undo multiple times after pasting figure", async () => {
+    setCellContent(model, "D6", "HELLO");
+    createTestChart("gauge");
+    await nextTick();
+    parent.env.model.dispatch("SELECT_FIGURE", { id: chartId });
+    await nextTick();
+
+    copy(model);
+    await simulateClick(".o-grid-overlay", 0, 0);
+    paste(model, "A1");
+    await nextTick();
+
+    const dispatchCtrlKey = (key: string) => {
+      document.activeElement?.dispatchEvent(
+        new KeyboardEvent("keydown", { key, ctrlKey: true, bubbles: true })
+      );
+    };
+
+    // CTRL+Z
+    dispatchCtrlKey("z");
```

Comment:
Your helper isn't really making a difference in length/readability VS `keyDown({ key: "z", ctrlKey: true });`. I'd remove it. The `// CTRL+Z` comments also look useless, don't put a comment if the code is easy to understand 🙂